all_model_data.json

```diff
@@ -985,5 +985,45 @@
       "type": "classification",
       "split": 5
     }
+  },
+  {
+    "model": "predModels.modelsplit",
+    "pk" : 1,
+    "fields": {
+      "model_path" : "model_cache\/split_1-model-classification.sav",
+      "estimator_path" : null,
+      "predtype" : "classification",
+      "type" : "noCluster"
+    }
+  },
+  {
+    "model": "predModels.modelsplit",
+    "pk" : 2,
+    "fields": {
+      "model_path" : "model_cache\/split_5-model-regression.sav",
```

Comment:
file doesn't exist

Reply:
that's strange, i'll remove it from all_data_model or is it better to create it?

Reply:
Better add it like you did with the classification file. But very good for sample data. But does it work? have you tried resetting the database and loading this file?

Reply:
With the loaddata doesn't give error

Reply:
No I'll recreate the regression model

Reply:
But is the data complete enough that everything works in the frontend?